package/gluon-web-wifi-config/i18n/gluon-web-wifi-config.pot

```diff
@@ -33,3 +33,20 @@ msgid ""
 "values include the antenna gain where available, but there are many devices "
 "for which the gain is unavailable or inaccurate."
 msgstr ""
+
+msgid "Outdoor installation"
+msgstr ""
+
+msgid "Node will be installed outdoors"
+msgstr ""
+
+msgid ""
+"Configuring the node for outdoor use tunes the 5 GHz radio to a frequency "
+"and transmission power that conforms with the local regulatory requirements. "
+"It also enables dynamic frequency selection (DFS; radar detection). At the "
+"same time, mesh functionality is disabled as it requires neighbouring nodes "
+"to stay on the same channel permanently."
+msgstr ""
+
+msgid "HT Mode"
+msgstr ""
```

Comment:
This translation section should be in a file in an independent package-specific location.